spec/controllers/cafe_works_controller_spec.rb

```diff
@@ -84,23 +84,23 @@
       context 'with no user' do
         it 'update worker' do
           patch :update_worker,
-                {
-                    id: cwork_access.to_param,
-                    cafe_work: attributes_for(:assignee, :test, access_code: cwork_access.access_code)
-                }
-          cwork_access.reload
+            {
+            id: cwork_access.to_param,
+            cafe_work: attributes_for(:assignee, :test, access_code: cwork_access.access_code)
+          }
+            cwork_access.reload
 
-          cwork_access.worker.attributes.should include(attributes_for(:assignee, :test))
+            cwork_access.worker.attributes.should include(attributes_for(:assignee, :test))
         end
 
         it 'redirects to the cafe_work' do
           patch :update_worker,
-                {
-                    id: cwork_access.to_param,
-                    cafe_work: attributes_for(:assignee, access_code: cwork_access.access_code)
-                }
+            {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.